analysis/runtime/src/events.rs

```diff
@@ -62,7 +60,10 @@ pub enum EventKind {
     StoreAddrTaken(Pointer),
 
     /// The pointer that appears as the address result of addr_of(Local)
-    AddrOfLocal(Pointer, Local),
+    AddrOfLocal(Pointer, Local, u32),
+
+    /// The address of a constant value
+    AddrOfConst(Pointer, u32),
```

Comment:
Can you add comments here explaining the meaning of the various fields?  I guess for `AddrOfLocal` it's `(resulting pointer value, local whose address was taken, size of that local)`?  And similarly for `AddrOfConst` but without an indication of the thing whose address was taken?